api/clients/accountant.go

```diff
@@ -8,15 +8,15 @@ import (
 	"sync"
 	"time"
 
-	commonpb "github.com/Layr-Labs/eigenda/api/grpc/common"
+	disperser_rpc "github.com/Layr-Labs/eigenda/api/grpc/disperser/v2"
 	"github.com/Layr-Labs/eigenda/core"
 	"github.com/Layr-Labs/eigenda/core/meterer"
 )
 
 var requiredQuorums = []uint8{0, 1}
 
 type Accountant interface {
-	AccountBlob(ctx context.Context, numSymbols uint64, quorums []uint8) (*commonpb.PaymentHeader, error)
+	AccountBlob(ctx context.Context, numSymbols uint64, quorums []uint8) (*core.PaymentMetadata, error)
 }
```

Comment:
Why is this interface defined if it's not being used in the disperser_client?

Reply:
`AccountBlob` is used in `DisperseBlob`, but I guess we wouldn't need the interface to access this function... I thought this is good practice, but happy to remove if that's not the case